src/openssl/ssl/context.cr

```diff
@@ -320,36 +320,27 @@ abstract class OpenSSL::SSL::Context
   end
 
   # Sets the current ciphers and ciphers suites to **modern** compatibility level as per Mozilla
-  # recommendations. See `CIPHERS_MODERN` and `CIPHER_SUITES_MODERN`. See `#security_level=` for some
-  # sensible system configuration.
+  # recommendations. See `#security_level=` for some sensible system configuration.
+  #
+  # Does nothing as per Crystal 1.13.
```

Comment:
Suggestion: mark it with `NOTE` or `WARNING` for better visibility.

```suggestion
  # WARNING: Does nothing as per Crystal 1.13.
```

ditto elsewhere